providers/default/digests/build.info

```diff
@@ -1,17 +1,17 @@
 SOURCE[../../../libcrypto]=\
-        null.c
+        null_prov.c
```

Comment:
Oh, right, I noticed that one yesterday...  why does it exist at all?

Note that this isn't meant to impact this PR, but is rather a heads up for another cleanup PR

Reply:
this is equivalent to EVP_MD null_md. I dont know if it is needed or not.